addons/print/models/ir_actions_print.py

```diff
@@ -64,6 +64,17 @@ def _run_action_print(self, eval_context=None):  # pylint: disable=unused-argume
                     active_id,
                 )
                 printer.spool_report(records.ids, report)
+                # Log a note in the audit trail
+                # of the affected records when a report is direct printed.
+                # Only affects models who have the class attribute AUDIT_LOG_PRINTING set to True.
+
+                # NB: If setting this class attribute on models,
+                # ensure the model has `_inherit = ["mail.thread"]`
+                if hasattr(obj, "AUDIT_LOG_PRINTING") and obj.AUDIT_LOG_PRINTING == True:
```

Comment:
Nit: could this just be
```python
hasattr(obj, "AUDIT_LOG_PRINTING") and obj.AUDIT_LOG_PRINTING
```
or if it must be a bool
```python
hasattr(obj, "AUDIT_LOG_PRINTING") and obj.AUDIT_LOG_PRINTING is True
```
?

Reply:
(same would apply to the other commit)

Reply:
Changed :)